platformio.ini

```diff
@@ -36,6 +36,7 @@ platform = teensy
 board = teensy35
 framework = arduino
 upload_protocol = teensy-cli
+test_build_src = true
```

Comment:
should be removed- deprecated

Reply:
This setting just enables building src code when invoking a "pio test" and doesn't affect pio build or pio run - not including this setting means any pio test in the teensy35 environment will fail

Is there a particular reason it was removed?